src/cloud/env/environment-vars_magento.md

```diff
@@ -32,3 +32,6 @@ If you need to change the Admin URL after installation, use the `magento-cloud v
 ```bash
 magento-cloud variable:set ADMIN_URL magento_A8v10 -e master
 ```
+
+{:.bs-callout-info}
+Use only letters (a-z or A-Z) or numbers (0-9) for a custom admin path. No spaces or other characters are allowed.
```

Comment:
```suggestion
The `ADMIN_URL` value accepts letters (a-z or A-Z) or numbers (0-9) for a custom admin path. Spaces or other characters are **not** accepted.
```

Reply:
Thank you, @hguthrie,

Will be fixed in the next commit.